src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpPreferStreamAsyncMemoryOverloads.Fixer.cs

```diff
@@ -38,5 +39,19 @@ public sealed class CSharpPreferStreamAsyncMemoryOverloadsFixer : PreferStreamAs
                 });
             }
         }
+
+        protected override bool IsSystemNamespaceImported(IReadOnlyList<SyntaxNode> importList)
+        {
+            foreach (SyntaxNode import in importList)
+            {
+                if (import is UsingDirectiveSyntax usingDirective &&
+                    usingDirective.Name is IdentifierNameSyntax identifierName &&
+                    identifierName.Identifier.Text == "System")
```

Comment:
```suggestion
                if (import is UsingDirectiveSyntax { Name: IdentifierNameSyntax { Identifier: { Text: "System" } } })
```

Reply:
Nice suggestion @CyrusNajmabadi, let's use the new stuff 😊.
I guess it shouls also be `nameof(System)` here, based on your other comment, right?